scionlab/tests/test_user_as_views.py

```diff
@@ -240,17 +247,19 @@ def test_get_create_form(self):
         self.app.set_user(TESTUSER_EMAIL)
         create_page = self.app.get(reverse('user_as_add'))
         form = create_page.form
-        self.assertEqual([o[2] for o in form['attachment_point'].options],
-                         self.attachment_points)
-        self.assertEqual(form['public_port'].value,
+        # Ignore the default value which is just "---------"
+        self.assertEqual(set(o[2] for o in form['form-0-attachment_point'].options if not o[1]),
+                         set(self.attachment_points))
+        self.assertEqual(form['form-0-public_port'].value,
                          str(DEFAULT_PUBLIC_PORT))
 
-    @parameterized.expand(UserASFormTests.valid_form_params)
-    def test_submit_create_form_valid(self, **kwargs):
+    @parameterized.expand(zip(filter(lambda c: len(c['attachments']) == 1,
+                                     UserASFormTests.valid_form_cases)))
```

Comment:
It shouldn't since the endpoint that we are testing, `user_as_add`, expects the user to only choose an attachment point (hence keeping the creation as simple as possible, assuming that this scenario will be the most common one)